### PR TITLE
Use Convars + Create fr.lua

### PR DESCRIPTION
**Pull request by TonybynMp4**

### CODE REVIEW

File: locales/fr.lua [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        ["tog_slots"] = "Active les emplacements de touches",
+        ["use_item"] = "Utilise l'objet dans l'emplacement ",
+    },
+    menu = {
+        ["vending"] = "Distributeur",
+        ["craft"] = "Fabriquer",
+        ["o_bag"] = "Ouvrir le Sac",
+    },
+    interaction = {
+        ["craft"] = "~g~E~w~ - Fabriquer",
```
Comment (xchopin) [inline locales/fr.lua:51]: Colors don't work you could remove them

Comment (TonybynMp4) [reply locales/fr.lua:51]: @ChatDisabled this true ?

Comment (ChatDisabled) [reply locales/fr.lua:51]: Not on my PC anymore but it displayed fine with the green E? Doubt that the other color won't work?

Comment (BerkieBb) [reply locales/fr.lua:51]: https://github.com/qbcore-framework/qb-inventory/blob/main/client/main.lua#L1080
They do still work

Comment (TonybynMp4) [reply locales/fr.lua:51]: image
looks like it works yeah

Comment (xchopin) [reply locales/fr.lua:51]: Great to hear it has been fixed recently, I was thinking of this PR when I wrote my comment https://github.com/qbcore-framework/qb-garages/pull/242

Comment (ChatDisabled) [reply locales/fr.lua:51]: Probably wrong use case and people got confused. But this one is good to go 👍🏼.

**Pull request closed without merging**